package.json

```diff
@@ -268,6 +268,7 @@
 		"@automattic/calypso-build": "file:./packages/calypso-build",
 		"@automattic/calypso-color-schemes": "file:./packages/calypso-color-schemes",
 		"@babel/plugin-transform-react-jsx": "7.3.0",
+		"@wordpress/babel-plugin-import-jsx-pragma": "2.1.0",
```

Comment:
Should this dependency be in `packages/calypso-build/package.json` since it's used by that package's Babel config rather than by root Babel config?

Reply:
To answer myself; `devDependencies` need to be in root package.json as per docs: https://github.com/Automattic/wp-calypso/blob/bbcdcc26144b2507e9e3876a42d06492cd37381f/docs/monorepo.md

Reply:
Uh yep, makes sense. Thanks, good spot!

Reply:
I would expect the babel plugin and transform to move _out_ of the root package.json and into this package's `dependencies`:

```json
{
"@babel/plugin-transform-react-jsx": "7.3.0",
"@wordpress/babel-plugin-import-jsx-pragma": "2.1.0"
}
```

Reply:
Same for [`@babel/plugin-transform-react-jsx`](https://github.com/Automattic/wp-calypso/pull/32024/files#diff-4ce8d7df7e4fc2d9cf86e3daddf99bc3R12) I think.

Reply:
Right, so anything `@automattic/calypso-build` relies on should be in its `dependencies` and `calypso-build` itself should be in consumer's `devDependencies`?

Reply:
> Right, so anything @automattic/calypso-build relies on should be in its dependencies

👍 Definitely

> and calypso-build itself should be in consumer's devDependencies?

👍 Probably